scala-mode2.el

```diff
@@ -169,5 +170,11 @@ When started, runs `scala-mode-hook'.
                '("\\.\\(scala\\|sbt\\)\\'" . scala-mode))
   (modify-coding-system-alist 'file "\\.\\(scala\\|sbt\\)\\'" 'utf-8))
 
+(defvar scala--prettify-symbols-alist scala-mode-pretty-all-alist)
+(defun scala-add-pretty ()
+  (setq prettify-symbols-alist scala--prettify-symbols-alist))
```

Comment:
should this `setq`rather be `add-to-list`? Is it good practice to override the existing list?